lib/hello/catalog/category.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Hello.Catalog.Category do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "categories" do
```

Comment:
@keliumJU what is a schema?

Reply:
A schema are structures that allow us to map model structures like %User{}, %Product{} to tables with fields in a database how postgres.